File: SampleLibrary/SampleLibrary/WordInfo.cs
namespace SampleLibrary
{
    public class WordInfo
    {
        public WordInfo(string fileName, int position)
        {
            FileName = fileName;
            Position = position;
        }

        public WordInfo()
        {
        }

        public int WordInfoId { get; set; }
        public string FileName { get; }
        public int Position { get; }
        public int WordId { get; set; }
        public Word Word { get; set; }
    }
}

File: Phase3/src/main/java/WordInfo.java
public class WordInfo {
    private String fileName;
    private int position;

    public WordInfo(String fileName, int position) {
        this.fileName = fileName;
        this.position = position;
    }

    public String getFileName() {
        return fileName;
    }

    public int getPosition() {
        return position;
    }
}
